src/test/resources/xtdb/sql/logic_test/direct-sql/identifier-case-insensitivity.test

```diff
@@ -71,19 +71,19 @@ cat
 query TI
 SELECT T1.col1, AVG(t1.col2) FROM t1 GROUP BY T1.col1
 ----
+30

```

Comment:
as @mbutlerw mentioned, there's a sorting construct within the SLT runner that's probably more suitable here